dashboard/modules/job/job_agent.py

```diff
@@ -0,0 +1,103 @@
+import aiohttp
+from aiohttp.web import Request, Response
+import dataclasses
+import json
+import logging
+import traceback
+
+import ray.dashboard.optional_utils as optional_utils
+import ray.dashboard.utils as dashboard_utils
+from ray.dashboard.modules.job.common import (
+    JobSubmitRequest,
+    JobSubmitResponse,
+)
+from ray.dashboard.modules.job.job_manager import JobManager
+from ray.dashboard.modules.job.utils import parse_and_validate_request, find_job_by_ids
+from ray.core.generated import gcs_service_pb2_grpc
+
+
+routes = optional_utils.ClassMethodRouteTable
+logger = logging.getLogger(__name__)
+
+
+class JobAgent(dashboard_utils.DashboardAgentModule):
+    def __init__(self, dashboard_agent):
+        super().__init__(dashboard_agent)
+        self._job_manager = None
+        self._gcs_job_info_stub = None
+
+    @routes.post("/api/job_agent/jobs/")
+    @optional_utils.init_ray_and_catch_exceptions()
+    async def submit_job(self, req: Request) -> Response:
+        result = await parse_and_validate_request(req, JobSubmitRequest)
+        # Request parsing failed, returned with Response object.
+        if isinstance(result, Response):
+            return result
```

Comment:
Print an error log here about what's happening.

Reply:
we print the error log in function `parse_and_validate_request`

Reply:
![image](https://user-images.githubusercontent.com/11995469/187857539-ef7469c3-7104-4d8a-ac38-904c8b07702b.png)